about.html

```diff
@@ -0,0 +1,194 @@
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="headline">
+                <h3>
+                    "Hello! African Summit!"
+                </h3>
+                <h1>
+                    PAN African Cloud Computing Summit 2022
+                </h1>
+                <p class="headline-description">
+                    The Global PAN African Summit brings together the community of experts, academics, and innovators who comprise come togter to build an impactful event   around  creating a positive change with speakers 
+                    from over 54 African nations and world beyond is talking place by October, in Nigeria.
+                </p>
+                <div class="headline-desktop">
+                    <p>The PAN African Global Summit brings together the community of
+                        experts, researches, and innovators who comprise the leading world leaders in the cloud computing domain. Their combined presense will usher a meeting pact with leadng figures in technology at large.
+                    </p>
+                    <p>
+                        In 2022, we made efforts to bring in the big guns and the most eligant actors off the cloud computing base. from Amazon, Google, Nvidia, Microsoft, CircleCI, Dropbox, Oracle etc.
+                    </p>
+                    <p>
+                        If your field of practive evolve around, the cloud, Big Data, IoT, Metaverse, Quantum Computing, SDN and any other gallery of next gneneration Technologies which we haven't mentioned, do well to be present with us this year 2022 and share in the deep knowledge of the cloud and beyound.
+
+                    </p>
+                </div>
+                <div class="headline-paragraph-contact">
+                    <p>
+                        Please contact us per Email for any further questions about PAN African Cloud Computing Summit 2022!
+                    </p>
+                    <a href="https://www.google.com">PANACCS2022@PANNigeria.org</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="container">
+        <h2 class="main-title">
+            PAN African Cloud Computing  Global Summit 2022 Logo
+        </h2>
+        <p class="main-paragraph">
+            The logo of PANACCS Summit 2022 was decided through the logo competition from 4th August to 7th September.
+        </p>
+        <p class="main-paragraph-desktop">
+            The logo of PANACCS Summit 2022 was decided through the logo competition from 4th August to 7th September.
+            online public voting and Selection Committee's judging process, the logo from Balandele Juwon won the prize.
+            After consultation with Mr. Nalandele we optimized the prize-winning-logo for this year's summit as following:
+        </p>
+        <div class="main-logo">
+            <img src="./Image/PAN_Logo.jpg" alt=" PANACCS Logo">
```

Comment:
[OPTIONAL] Kindly fix the image URL, it should be `Images` instead of `Image` 🤗

![image](https://user-images.githubusercontent.com/73714615/190589810-17f4ad60-da8d-44e8-a840-8ab4b78406a7.png)